model/src/pyrenew/regression.py

```diff
@@ -114,38 +138,54 @@ def predict(
         coefficients : ArrayLike
             Sampled prediction coefficients distribution generated
             from coefficients priors.
+        predictor_values : ArrayLike(n_predictors, n_observations)
+            Matrix of predictor variables (covariates) for the
+            regression problem. Each row should represent the
+            predictor values corresponding to an observation;
+            each column should represent a predictor variable.
+            You do not include values of 1 for the intercept;
+            these will be added automatically.
 
         Returns
         -------
         ArrayLike
             Array of transformed predictions.
         """
-        transformed_prediction = (
-            intercept + self.fixed_predictor_values @ coefficients
-        )
+        transformed_prediction = intercept + predictor_values @ coefficients
         return self.transform.inv(transformed_prediction)
 
-    def sample(self) -> dict:
+    def sample(self, predictor_values: ArrayLike) -> dict:
```

Comment:
```suggestion
    def sample(self, predictor_values: ArrayLike) -> GLMPredictionSample:
```

Reply:
changed in cf3b96a7f790b891ec1df0141542bcb9863dd365